app/presenters/publishing_api/role_presenter.rb

```diff
@@ -52,12 +52,24 @@ def polymorphic_path
       end
     end
 
+    def whip
+      return {} unless item.whip_organisation_id != nil
```

Comment:
There's a double negative here, and the code would probably be easier to read if this was avoided.